src/test/java/in/zapr/druid/druidry/virtualColumn/ExpressionVirtualColumnTest.java

```diff
@@ -0,0 +1,45 @@
+package in.zapr.druid.druidry.virtualColumn;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import in.zapr.druid.druidry.dimension.enums.OutputType;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ExpressionVirtualColumnTest {
```

Comment:
Can you also write negative test case when required field/s is/are not set. You can look at other test classes for reference.